apps/researcher/src/app/[locale]/communities/settings.ts

```diff
@@ -0,0 +1 @@
+export const limit = 12;
```

Comment:
The word `limit` is quite generic and its meaning may not be clear - what kind of limit is it? Would it be an idea to rename it and make its meaning explicit?